src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -631,6 +631,78 @@ mongoc_client_encryption_rewrap_many_datakey (
 }
 
 
+bool
+mongoc_client_encryption_delete_key (
+   mongoc_client_encryption_t *client_encryption,
+   const bson_value_t *keyid,
+   bson_t *reply,
+   bson_error_t *error)
+{
+   bson_init (reply);
+   return _disabled_error (error);
+}
+
+
+mongoc_cursor_t *
+mongoc_client_encryption_get_key (mongoc_client_encryption_t *client_encryption,
+                                  const bson_value_t *keyid,
+                                  bson_error_t *error)
+{
+   _disabled_error (error);
+   return NULL;
+}
+
+
+mongoc_cursor_t *
+mongoc_client_encryption_get_keys (
+   mongoc_client_encryption_t *client_encryption, bson_error_t *error)
+{
+   _disabled_error (error);
+   return NULL;
+}
+
+
+bool
+mongoc_client_encryption_add_key_alternate_name (
+   mongoc_client_encryption_t *client_encryption,
+   const bson_value_t *keyid,
+   const char *keyaltname,
+   bson_value_t *key_doc,
+   bson_error_t *error)
+{
+   if (key_doc) {
+      memset (key_doc, 0, sizeof (*key_doc));
```

Comment:
I wasn't sure if these were initialized when commenting on the documentation, but it appears this parameter is optional. Should that be documented? If it isn't meant to be optional, should you assert the parameter is non-null?

Reply:
Yes, out parameters are meant to be optional. Will fix documentation and related cases.